ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/config/tls/client/PicoCliClientTlsOptions.java

```diff
@@ -43,6 +58,13 @@
 
   @Override
   public Optional<ClientTlsTrustOptions> getTrustOptions() {
+    // don't validate here just send options through if present
+    if (caAuthEnabled && knownServersFile == null) {
+      return Optional.empty();
+    }
+    PicoCliClientTlsTrustOptions trustOptions = new PicoCliClientTlsTrustOptions();
```

Comment:
nit: thinking we shouldn't be hand-making these ... just return the embedded object?

Reply:
fixed